mobile/lib/screens/kya/kya_lessons_page.dart

```diff
@@ -90,29 +88,44 @@ class _KyaLessonsPageState extends State<KyaLessonsPage> {
         color: CustomColors.appBodyColor,
         child: Column(
           children: [
-            const Spacer(),
+            Visibility(
+              visible: currentIndex == 9,
```

Comment:
@Mnoble-19 this looks hardcoded. What happens when we add 1 lesson to the KYA, totalling up to 11 lessons.